### ISSUE TITLE
Issues following getting started documentation

### ISSUE THREAD

**Issue by sibille** [microsoft/react-native-windows#5033]
### Environment

1. `react-native -v`: 0.61.5
2. `react-native run-windows --info`: 0.61.10
3. `node -v`: v12.9.1
4. `npm -v`: 6.14.1
5. `yarn --version`:1.22.0
6. VS Code 1.45.1
7. React Native Tools Extension: 0.16.0
8. OS: Windows 10 Version 1909

### Steps to Reproduce

Following the getting started documentation from https://microsoft.github.io/react-native-windows/docs/getting-started we had the following issues:

- From command line: 
When running "npx react-native-windows-init –overwrite" we sometimes get an “Failed to generate Self-signed certificate. Using Default Certificate. Use Visual Studio to renew it.” warning.
Not sure what causes this, but it should be displayed as an error because the rest of the process fails unless the certificate is renewed. 

- From VSCode:
We only could start the generated application successfully when running VSCode in Admin Mode. 
Is this expected? We're also getting the following error, although compilation continues afterwards and the application starts fine in the end:
```
jest-haste-map: watch error:
  Error: EBUSY: resource busy or locked, open 'c:\temp\w16\msbuild.ProjectImports.zip'
    at Object.openSync (fs.js:448:3)
    at Object.readFileSync (fs.js:350:35)
    at getContent (c:\temp\w16\node_modules\jest-haste-map\build\worker.js:149:41)
    at Object.<anonymous> (c:\temp\w16\node_modules\jest-haste-map\build\worker.js:199:9)
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (c:\temp\w16\node_modules\jest-haste-map\build\worker.js:78:24)
    at _next (c:\temp\w16\node_modules\jest-haste-map\build\worker.js:98:9)
    at c:\temp\w16\node_modules\jest-haste-map\build\worker.js:103:7
    at new Promise (<anonymous>)
    at Object.<anonymous> (c:\temp\w16\node_modules\jest-haste-map\build\worker.js:95:12)
```

**Comment by asklar** [comment]
The elevation issues should be much less severe/common in 0.63. I'm not able to reproduce the msbuild.ProjectImports.zip issue but I do remember we saw it at one point in earlier releases, so this might not be an issue in 0.63 :)

**Comment by sibille** [comment]
Awesome! How can I create project using 0.63?

**Comment by asklar** [comment]
you can try the 0.62 steps here:
https://microsoft.github.io/react-native-windows/docs/next/getting-started

if you want the latest and greatest (what will eventually become 0.63, 64, 65, etc.) then clone the RNW repo locally and follow the steps in https://github.com/microsoft/react-native-windows/wiki/Contributing-to-the-CLI
specifically, you'd create the app as you do today, then add windows with `node <root>\node_modules\react-native-windows-init\bin.js --language cpp --version file:<root>\vnext`

**Comment by chrisglein** [comment]
We believe this is fixed in 0.62. If that's not the case let us know.

**Comment by sibille** [comment]
Ok, will test and let you know.

**Comment by asklar** [comment]
for the ProjectImports.zip thing I have an explanation of why we hit it even though we don't use zip files. I'll fix this.

**Comment by sibille** [comment]
> We believe this is fixed in 0.62. If that's not the case let us know.

Works fine in 0.62.
